### PR TITLE
[FEAT] Adding short descriptions to the app page.

### PR DESCRIPTION
**Pull request by simonsasse**

### CODE REVIEW

File: _apps/delly.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: <b>Delly</b> - Integrated Structural Variant Discovery
+title: Delly
+subtitle: Integrated Structural Variant Discovery
```
Comment (eseiler) [inline _apps/delly.md:3]:
```suggestion
subtitle: Integrated structural variant discovery
```

File: _apps/igenvar.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
-title: iGenVar - An SV, Indel, and SNP Caller using short and long reads
+title: iGenVar
+subtitle: An SV, Indel, and SNP Caller using short and long reads
```
Comment (eseiler) [inline _apps/igenvar.md:3]:
```suggestion
subtitle: An SV, indel, and SNP caller using short and long reads
```

File: _apps/lambda.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: LAMBDA
+subtitle:  A local aligner optimized for many query sequences and searches in protein space
```
Comment (eseiler) [inline _apps/lambda.md:3]:
```suggestion
subtitle: A local aligner optimized for many query sequences and searches in protein space
```

File: _apps/slimm.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
-title: SLIMM - Species Level Identification of Microbes from Metagenomes
+title: SLIMM
+subtitle: Species Level Identification of Microbes from Metagenomes
```
Comment (eseiler) [inline _apps/slimm.md:3]:
```suggestion
subtitle: Species level identification of microbes from metagenomes
```

File: _apps/vaquita.md [hunk 5]
```diff
@@ -1,11 +1,12 @@
 ---
-header:
-  app_image: /assets/images/apps/vaquita_logo.png
-title: Vaquita, fast and Accurate Identification of Structural Variation Using Combined Evidence
+title: Vaquita
+subtitle: Fast and Accurate Identification of Structural Variation Using Combined Evidence
```
Comment (eseiler) [inline _apps/vaquita.md:3]:
```suggestion
subtitle: Fast and accurate identification of structural variation using combined evidence
```

**Review by eseiler — CHANGES_REQUESTED** [review]
Looks good, I tried to unify the capitalisation

(Also needs rebase)

**Comment by eseiler** [comment]
And maybe add `Optimal string similarity search and join` as subtitle for `String Similarity Search/Join` since it the only one without?

**Comment by eseiler** [comment]
Now the badge is on a new line, can you make it on the same line again?

**Comment by simonsasse** [comment]
I'm on it :) Unluckily my Jekyll is messed up, I thought it'd ne fixed faster. I'm gonna repair my Jekyll and then this PR :)

**Review by eseiler — APPROVED** [review]

**Pull request merged**